README.md

```diff
@@ -8,6 +8,10 @@ Please [file an issue](https://github.com/WebKit/standards-positions/issues/new)
 
 ### Possible positions
 
+**Please note:** Only once a label is assigned to an issue does it represent WebKit's position.
```

Comment:
“Note” is a term-of-art in specs, and I think has the opposite meaning there than what you intend to mean here. I’d recommend removing the “please note” text.

Reply:
Done.